struct.go

```diff
@@ -128,6 +128,31 @@ func Field(fieldPtr interface{}, rules ...Rule) *FieldRules {
 	}
 }
 
+// FindStructFieldJSONName gets the value of the `json` tag for the given field in the given struct
+func FindStructFieldJSONName(structPtr interface{}, fieldPtr interface{}) (string, error) {
+	value := reflect.ValueOf(structPtr)
+	if value.Kind() != reflect.Ptr || !value.IsNil() && value.Elem().Kind() != reflect.Struct {
+		// must be a pointer to a struct
+		return "", NewInternalError(ErrStructPointer)
+	}
+	if value.IsNil() {
+		// treat a nil struct pointer as valid
+		return "", nil
+	}
+	value = value.Elem()
+
+	fv := reflect.ValueOf(fieldPtr)
+	if fv.Kind() != reflect.Ptr {
+		// must be a pointer to a field
+		return "", NewInternalError(ErrFieldPointer(0))
+	}
+	ft := findStructField(value, fv)
+	if ft == nil {
+		return "", NewInternalError(ErrFieldNotFound(0))
+	}
+	return getErrorFieldName(ft), nil
```

Comment:
Since the value this function returns might not come from the JSON tag (the `ErrorTag` [variable](https://github.com/jellydator/validation/blob/master/validation.go#L51) lets the user change the default 'json' value), this new function should probably be renamed as well. Perhaps `ErrorFieldName(struct, field)` would suffice?

Reply:
fixed

Reply:
Is there a chance that you forgot to push your changes? :thinking:

Reply:
everything should be fixed now